cli/spark/endpoints/cluster/cluster_ssh.py

```diff
@@ -52,19 +57,23 @@ def execute(args: typing.NamedTuple):
         connect=args.connect
     )
 
-    http_prefix = 'http://localhost:'
     log.info("-------------------------------------------")
     log.info("spark cluster id:    %s", ssh_conf.cluster_id)
     log.info("open webui:          %s%s", http_prefix, ssh_conf.web_ui_port)
     log.info("open jobui:          %s%s", http_prefix, ssh_conf.job_ui_port)
-    log.info("open jobhistoryui:   %s%s", http_prefix, ssh_conf.job_history_ui_port)
-    log.info("open jupyter:        %s%s", http_prefix, ssh_conf.jupyter_port)
-    log.info("open namenodeui:     %s%s", http_prefix, ssh_conf.name_node_ui_port)
-    log.info("open rstudio server: %s%s", http_prefix, ssh_conf.rstudio_server_port)
+    log.info("open jobhistoryui:   %s%s", http_prefix,
+             ssh_conf.job_history_ui_port)
+    log.info("open namenodeui:     %s%s",
+             http_prefix, ssh_conf.name_node_ui_port)
+    log.info("open rstudio server: %s%s", http_prefix,
+             ssh_conf.rstudio_server_port)
     log.info("ssh username:        %s", ssh_conf.username)
     log.info("connect:             %s", ssh_conf.connect)
     log.info("-------------------------------------------")
 
+    print_plugin_ports(cluster)
```

Comment:
I feel like there isn't a good reason to split plugins out into their own section like this, just print them like normal, but only show the ones that are actually installed on the cluster.